qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -188,12 +188,19 @@ def process_sample_template(self, study, user, callback):
         msg_level = "success"
         is_mapping_file = looks_like_qiime_mapping_file(fp_rsp)
 
+        if is_mapping_file and data_type == "":
```

Comment:
Yes, it is always a string since that is what tornado returns. The "non selected" value is coded as an empty string `""`, I think @wasade proposal can be done as I think that is the standard way of checking against an empty string.